.gitignore

```diff
@@ -0,0 +1,375 @@
+# Created by https://www.toptal.com/developers/gitignore/api/hugo,python,pycharm+all,visualstudiocode,kate,emacs,vim
```

Comment:
Half of this doesn't really apply to this repo. I'd rather have a cherry-picked set of ignores than ignoring every eventuality "just because".

For the time being I added Hugo's outputs to .gitignore. Personal configuration can also be written to a global ignores file, which git typically reads from .config/git/ignore (see also core.excludesFile).